test/e2e/diagram.overlays.rendering.test.ts

```diff
@@ -45,33 +44,72 @@ class ImageSnapshotThresholds extends MultiBrowserImageSnapshotThresholds {
   getFirefoxThresholds(): Map<string, ImageSnapshotThresholdConfig> {
     return new Map<string, ImageSnapshotThresholdConfig>([
       [
-        'overlays.start.flow.task',
+        'overlays.start.flow.task.gateway',
         {
-          linux: 0.0023, // 0.22444374007244416%
-          macos: 0.0025, // 0.24420815130801188%
-          windows: 0.007, // 0.6915678561497107%
+          linux: 0.0053, // 0.5229417116423329%
+          macos: 0.0061, // 0.6026399523082704%
         },
       ],
     ]);
   }
 }
 
+function buildOverlaySnapshotDir(config: MatchImageSnapshotOptions, position: OverlayPosition): string {
+  return join(config.customSnapshotsDir, `on-position-${position}`);
+}
+
+async function addOverlay(bpmnContainerElementHandle: ElementHandle<Element>, bpmnElementId: string, position: OverlayPosition): Promise<void> {
+  await page.fill('#bpmn-id-input', bpmnElementId);
+  await clickOnButton(position);
+  await bpmnContainerElementHandle.waitForSelector(`svg > g > g:nth-child(3) > g[data-bpmn-id="${bpmnElementId}"]`);
```

Comment:
⚠️ I would remove this assertion/expect. If there is an issue with the overlay add code, the svg won't be produced and then, the `waitForSelector` will timeout. So the test will stop here.
We are doing visual tests here, so we expect not seeing the overlay in that case, and a failure during the image diff.
This DOM check is then redundant.

In addition, we already have DOM integration tests that check svg generation so we test it twice and the code to perform the DOM selector is currently duplicated between this test and the code here.

Reply:
Maybe, we should only add **delay** calling to be sure that MxGraph redraw the container.

Reply:
yes you are right, it is enough to verify it in dom test

Reply:
We decided to remove it until the generated screenshot is ok.
If someday we have a bad screenshot, we can add a event listener, because MxGraph genereate event at overlay creation & deletion.